### PR TITLE
Add cli query for host and controller params

### PR DESCRIPTION
**Pull request by colin-axner**
## Description

closes: #569 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: modules/apps/27-interchain-accounts/client/cli/cli.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	controllercli "github.com/cosmos/ibc-go/v2/modules/apps/27-interchain-accounts/controller/client/cli"
+	hostcli "github.com/cosmos/ibc-go/v2/modules/apps/27-interchain-accounts/host/client/cli"
+)
+
+// GetQueryCmd returns the query commands for the interchain-accounts submodule
+func GetQueryCmd() *cobra.Command {
+	icaQueryCmd := &cobra.Command{
+		Use:                        "interchain-accounts",
+		Aliases:                    []string{"ica"},
+		Short:                      "interchain-accounts controller subcommands",
```
Comment (colin-axner) [inline modules/apps/27-interchain-accounts/client/cli/cli.go:15]:
```suggestion
		Short:                      "interchain-accounts subcommands",
```

File: modules/apps/27-interchain-accounts/controller/client/cli/query.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+)
+
+// GetCmdParams returns the command handler for the controller submodule parameter querying.
+func GetCmdParams() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "params",
+		Short:   "Query the current interchain-accounts controller submodule parameters",
+		Long:    "Query the current interchain-accounts controller submodule parameters",
+		Args:    cobra.NoArgs,
+		Example: fmt.Sprintf("%s query ibc-transfer params", version.AppName),
```
Comment (damiannolan) [inline modules/apps/27-interchain-accounts/controller/client/cli/query.go:21]:
```suggestion
		Example: fmt.Sprintf("%s query interchain-accounts controller params", version.AppName),
```

File: modules/apps/27-interchain-accounts/host/client/cli/query.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+)
+
+// GetCmdParams returns the command handler for the host submodule parameter querying.
+func GetCmdParams() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "params",
+		Short:   "Query the current interchain-accounts host submodule parameters",
+		Long:    "Query the current interchain-accounts host submodule parameters",
+		Args:    cobra.NoArgs,
+		Example: fmt.Sprintf("%s query ibc-transfer params", version.AppName),
```
Comment (damiannolan) [inline modules/apps/27-interchain-accounts/host/client/cli/query.go:21]:
```suggestion
		Example: fmt.Sprintf("%s query interchain-accounts host params", version.AppName),
```

**Review by damiannolan — APPROVED** [review]
Beautiful. Left two suggestions for updating example CLIs

**Review by seantking — APPROVED** [review]

**Pull request merged**